### PR TITLE
Update errors-inbox.mdx (Publish ~May 9, 2022)

### PR DESCRIPTION
**Pull request by meda123**
Bulk actions will be released to prod in a couple of days. Not sure if it should be part of the PR yet, a screenshot showing bulk actions' UI?

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

File: src/content/docs/errors-inbox/errors-inbox.mdx [hunk 1]
```diff
@@ -102,6 +102,10 @@ Your errors inbox displays the total number of occurrences of each error group w
 
 Using the dropdown in the top right, you can sort the list of grouped errors by the number of occurrences or by the error that was last seen (latest first).
 
+### Bulk Actions 
+
+You can update the status (e.g. resolve, ignore, unresolve) of multiple error groups, up to 2,000, all at once. In the inbox view, you can select the Error group checkbox and update all error groups in the inbox. You can also select individual error groups and update their status.
```
Comment (paperclypse) [inline src/content/docs/errors-inbox/errors-inbox.mdx:107]:
```suggestion
You can update the status (for example, resolve, ignore, unresolve) of multiple error groups (up to 2,000) all at once. In the inbox view, select the **Error group** checkbox to update all of your error groups in the inbox. You can also select individual error groups and update their statuses.
```

**Review by paperclypse — DISMISSED** [review]
Looks good!

**Review by paperclypse — APPROVED** [review]
Looks good!

**Comment by paperclypse** [comment]
Please check with @meda123 before merging this in.

**Comment by rhetoric101** [comment]
We were advised today that this will probably be ready for publication on May 9th.

**Pull request merged**